### PR TITLE
IO-1 Extending Command Manager window

### PR DESCRIPTION
**Pull request by Miretpl**
Patryk Janowski
Przemysław Mirowski

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java [hunk 1]
```diff
@@ -68,6 +70,14 @@ public synchronized void clearCurrentCommand() {
         currentCommand = null;
     }
 
+    public synchronized void runCurrentCommand() {
+        currentCommand.execute(driverManager.getCurrentDriver());
+    }
+
+    public synchronized void setCurrentDriver(DriverManager driverManager) {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java]: setCurrentDriver -> setDriverManager

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java [hunk 2]
```diff
@@ -116,5 +115,5 @@ public void run() {
             }
         });
     }
-
+
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java]: only whitespace modifications. Revert this file

**Review by kowallus — CHANGES_REQUESTED** [review]
Nice. Only minor issues.

File: src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java (hunk 1 above)
Comment (raionsakana) [reply src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java]: Done

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java (hunk 2 above)
Comment (raionsakana) [reply src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java]: Done

**Pull request merged**